io.weldr.welder-web.metainfo.xml

```diff
@@ -0,0 +1,15 @@
+<component type="addon">
+  <id>io.weldr.welder-web.metainfo.xml</id>
```

Comment:
`io.weldr.welder-web`

Reply:
That's exactly what is here, or I misunderstand you.

Note that welder-web currently can build a welder-web.rpm (with this file) or a cockpit-composer.rpm (with the other one). This is also why I asked whether we can drop one name now. But for now this PR supports both.

Reply:
I would drop the `metainfo.xml` part, but I can't say I am an export on AppStream ids.